Neos.Flow/Tests/Functional/Http/Fixtures/Controller/FooController.php

```diff
@@ -11,16 +11,19 @@
  * source code.
  */
 
+use Neos\Flow\Mvc\ActionResponse;
 use Neos\Flow\Mvc\Controller\AbstractController;
 
 class FooController extends AbstractController
```

Comment:
```suggestion
class FooController implements ControllerInterface
```

if we override everything there is not much gain

Reply:
i was dumb, we use `initializeController` 😂  sorry reverted.